### PR TITLE
[#MH2018-29] [2] Sh feature/special offers banner

### PR DESCRIPTION
**Pull request by IsmaelVillavicencio**
Originals
desktop 1024
mobile portrait
How it looks!!
Link:
https://fiorella-dev.myshopify.com/

### CODE REVIEW

File: fiorella/src/styles/layout/_special-offers-banner.scss [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    right: 5px;
+  }
+
+  .offers-title-icon {
+    vertical-align: middle;
+    width: 40px;
+  }
+}
+
+.offers-message-box{
```
Comment (magmahound) [inline fiorella/src/styles/layout/_special-offers-banner.scss]: Opening curly brace `{` should be preceded by one space

File: fiorella/src/styles/layout/_special-offers-banner.scss [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+    }
+
+    @media #{$desktop} {
+      font-size: 40px;
+      letter-spacing: 4px;
+      line-height: 60px;
+      width: 450px;
+    }
+
+    .offers-price {
```
Comment (magmahound) [inline fiorella/src/styles/layout/_special-offers-banner.scss]: Selector should have depth of applicability no greater than 2, but was 3

File: fiorella/src/sections/special-offers-banner.liquid [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+      <p class="offers-description">{{ section.settings.offers_description }}
+        <span class="offers-price">{{ section.settings.offers_price | money_without_trailing_zeros }}</span>
+      </p>
+    </div>
+  </div>
+</section>
+
+{% schema %}
+{
+  "name":"Special offers banner",
```
Comment (jaimelr) [inline fiorella/src/sections/special-offers-banner.liquid]: There is one space missing after the colon `:`

File: fiorella/src/sections/special-offers-banner.liquid [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+    },
+    {
+      "type":"image_picker",
+      "id":"offers_img_background_mobile",
+      "label":"background image mobile"
+    },
+    {
+      "type":"image_picker",
+      "id":"offers_img_background_desktop",
+      "label":"background image desktop"
```
Comment (jaimelr) [inline fiorella/src/sections/special-offers-banner.liquid]: There is one space missing after the colon `:`

File: fiorella/src/sections/special-offers-banner.liquid [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+
+<style type="text/css">
+.offers-box {
+  background-image: url({{ section.settings.offers_img_background_mobile | img_url: '800x' }});
+  background-size: auto 280px;
+  background-position: right;
+  background-repeat: no-repeat;
+}
+
+@media only screen and (min-width: 768px) {
```
Comment (jaimelr) [inline fiorella/src/sections/special-offers-banner.liquid]: This could be a SASS variable.

**Pull request merged**